src/components/POLine/POLineForm.js

```diff
@@ -43,75 +40,65 @@ import { ItemForm } from './Item';
 import { OtherForm } from './Other';
 import {
   ACCORDION_ID,
+  INITIAL_SECTIONS,
   MAP_FIELD_ACCORDION,
+  POL_TEMPLATE_FIELDS_MAP,
 } from './const';
 import getMaterialTypesForSelect from '../Utils/getMaterialTypesForSelect';
 import getIdentifierTypesForSelect from '../Utils/getIdentifierTypesForSelect';
 import getContributorNameTypesForSelect from '../Utils/getContributorNameTypesForSelect';
 import getOrderTemplatesForSelect from '../Utils/getOrderTemplatesForSelect';
 import { ifDisabledToChangePaymentInfo } from '../PurchaseOrder/util';
+import getOrderTemplateValue from '../Utils/getOrderTemplateValue';
 import calculateEstimatedPrice from './calculateEstimatedPrice';
-import asyncValidate from './asyncValidate';
-import validate from './validate';
-
-class POLineForm extends Component {
-  static propTypes = {
-    initialValues: PropTypes.object,
-    handleSubmit: PropTypes.func.isRequired,
-    stripes: stripesShape.isRequired,
-    onSubmit: PropTypes.func.isRequired,
-    onCancel: PropTypes.func,
-    order: PropTypes.object.isRequired,
-    pristine: PropTypes.bool,
-    submitting: PropTypes.bool,
-    parentResources: PropTypes.object,
-    change: PropTypes.func,
-    dispatch: PropTypes.func,
-    vendor: PropTypes.object,
-    isSaveAndOpenButtonVisible: PropTypes.bool,
-    enableSaveBtn: PropTypes.bool,
-  };
+import validateFundDistribution from './validate';
 
-  constructor(props) {
-    super(props);
-    this.state = {
-      sections: {
-        [ACCORDION_ID.lineDetails]: true,
-        [ACCORDION_ID.costDetails]: true,
-        [ACCORDION_ID.vendor]: true,
-        [ACCORDION_ID.eresources]: true,
-        [ACCORDION_ID.itemDetails]: true,
-        [ACCORDION_ID.other]: true,
-        [ACCORDION_ID.physical]: true,
-        [ACCORDION_ID.fundDistribution]: true,
-        [ACCORDION_ID.location]: true,
-      },
-    };
-  }
+const GAME_CHANGER_FIELDS = ['isPackage', 'orderFormat', 'checkinItems', 'packagePoLineId'];
```

Comment:
these fields controls other section to show, so we have to change them first, and fill other fields from order template later